ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1679,6 +1679,13 @@ literals:
 ## New fields for seismic detector
 - seismic_alarm
 
+## New fields for chilled beams
+- zone_air_occupied_temperature_setpoint
+- zone_air_unoccupied_cooling_temperature_setpoint
+- perimeter_heating_water_valve_percentage_sensor
```

Comment:
will this ever exist independently of another valve? 

if we know the device lives on a perimeter then would heating_water_valve_percentage_... be sufficient?

Reply:
Potentially there could be other heating valves, like reheat or four pipe chilled beams.